### PR TITLE
Updating UI components

### PR DESCRIPTION
**Pull request by SanjanaSogimatt**
### Changes:

- Updated header.astro file to reflect the new layout and components.
- Adjusted the Logo component for consistent rendering across devices.
- Modified Tailwind CSS classes to improve the design and responsiveness of the header.

## ✅ Type of Change
- [ ] 🎨 **UI/UX** (changes that affect the user interface)

**Updated version:**
Screenshot 2024-09-20 201931

## Summary by CodeRabbit

- **New Features**
	- Updated menu structure for a more streamlined navigation experience, including renaming "How It Works" to "Programs" and adding "Partnership."
	- Introduced a new `FeatureCard` component for enhanced feature presentation in a grid layout.

- **Improvements**
	- Enhanced the layout and styling of the `MemberCard` component for better visual presentation and responsiveness.
	- Increased logo height and updated footer description for improved branding.
	- Updated various dependencies to their latest versions for better performance and security.
	- Changed footer background color for a fresher look.
	- Added new color palette to enhance design options.

- **Bug Fixes**
	- Removed outdated menu items for a cleaner interface.
	- Eliminated sign-up links from the header for a simplified navigation.

### CODE REVIEW

File: src/layouts/function-components/FeatureCard.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+
+const FeatureCard = ({ feature_list, cardHeight = "h-[250px]", cardShadow = "shadow-sm" , background = "bg-white" }) => {
```
Comment (drtechie) [inline src/layouts/function-components/FeatureCard.jsx:2]: I would prefer camelCase variables in JS.
Some consistency will be good.

File: src/layouts/shortcodes/Button.jsx [hunk 2]
```diff
@@ -1,14 +1,24 @@
...
   return (
     <a
       href={href}
       target="_blank"
-      rel={`noopener noreferrer ${
-        rel ? (rel === "follow" ? "" : rel) : "nofollow"
+      rel={rel}
+      className={`inline-block px-5 py-3 rounded transition-all ${
+        buttonStyles[style]
       }`}
```
[the remark this answers is not in the document]
Comment (drtechie) [reply src/layouts/shortcodes/Button.jsx:21]: @SanjanaSogimatt check this comment.

File: src/layouts/shortcodes/Button.jsx [hunk 3]
```diff
@@ -1,14 +1,24 @@
...
+import React from "react";
+
+const Button = ({ 
+  href = "#", 
+  style = "secondary", 
+  children = "Click Me", 
+  rel = "noopener noreferrer" 
+}) => {
+  const buttonStyles = {
+    primary: "bg-white text-black border border-black hover:bg-black hover:text-white",
```
Comment (drtechie) [inline src/layouts/shortcodes/Button.jsx:10]: @SanjanaSogimatt 
this is fine. But it also assumes that primary and secondary buttons have certain colours.
Do you think we should finalize the colours and branding first? Editing these later is going to be a hassle.

File: src/layouts/shortcodes/Button.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (SanjanaSogimatt) [reply src/layouts/shortcodes/Button.jsx:21]: I'll update it

**Comment by SanjanaSogimatt** [comment]
@drtechie can you go through the pr once?

**Review by drtechie — APPROVED** [review]

**Pull request merged**